wagtail_localize/static_src/common/components/Section/index.tsx

```diff
@@ -40,27 +40,32 @@ const Title = styled.div`
         top: 0;
         line-height: 1.8em;
         left: 0;
-        width: 50px;
         color: #fff;
         padding: 0;
         margin: 0;
-        background-color: #f37e77;
+        background-color: var(--w-color-primary);
```

Comment:
would be good to fully hide the arrow in a purple box bit (next to "English to French translation")

<img width="644" alt="Screenshot 2022-08-29 at 13 23 58" src="https://user-images.githubusercontent.com/31622/187200526-c6edc110-bb04-4484-b9e1-14d0d713bf0e.png">

as it will bring the UI a tad closer to the rest of 4.0

Reply:
It strikes me as a slightly arbitrary UI element.

<img width="745" alt="Screenshot 2022-08-30 at 16 31 50" src="https://user-images.githubusercontent.com/237556/187478882-b876f5f0-10bc-44b2-8d24-f34301a8018c.png">

What if we just removed it entirely?

Reply:
<img width="950" alt="Screenshot 2022-08-30 at 16 35 01" src="https://user-images.githubusercontent.com/237556/187479986-d3aaedc0-7a33-4701-8e66-6ea0edc0e4fa.png">
<img width="949" alt="Screenshot 2022-08-30 at 16 35 12" src="https://user-images.githubusercontent.com/237556/187479981-c9e70733-a6f2-404c-92d9-1cdb35f990d1.png">
<img width="949" alt="Screenshot 2022-08-30 at 16 34 45" src="https://user-images.githubusercontent.com/237556/187479990-225f845e-ae92-46bd-aa89-2a51d36aba2e.png">

Reply:
For now I've simply removed the red/arrow thing as requested, leaving it looking like screenshot 2 (middle) from the comment above.

Reply:
I think removing the red arrow works in this first instance. Thank you